lib/tasks/canonical_models/canonical_materials.json

```diff
@@ -172,11 +181,11 @@
   },
   {
     "attributes": {
-      "name": "Small Antlers",
-      "item_code": "0006BC0B",
+      "name": "Troll Skull",
```

Comment:
I guess this one was added too since it doesn't appear to have been removed from elsewhere in the file during sorting.